cfgov/v1/preprocessed/css/molecules/image-text-50-50.less

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+   cfgov-refresh
+   Image and Text 50-50 molecule
+   ========================================================================== */
+
+.m-image-text-50-50 {
+    position: relative;
+    padding-bottom: 60px;
```

Comment:
Do we ever convert to anything other than ems? Maybe that does make sense within our project (not sure about all of CF).